src/threatmodel.adoc

```diff
@@ -49,287 +53,815 @@ utilized by this adversary.
-priority or masking | Supervisor Domains (Smsdia) w/ AIA
-
-| Secure Timestamp | Trusted timestamps | Required | Confidential supervisor
-domain qualifier for CSR accesses | Ensure TEE have consistent timestamp view |
-Supervisor Domains (Sdid)
-
-| Debug & Profile | Trusted performance monitoring unit data | Required |
-Confidential supervisor domain context switch of perf. mon. counters  |
-Ensure TEEs get correct PMU info; prevent data leakage due to PMU information
-(fingerprint attacks) | Supervisor Domains (Secure Debug)
-
-| Debug & Profile | Secure External Debug support | Required | Confidential
-supervisor domain qualifier for External debug controls | Support debug trigger
-registers for TVM | Supervisor Domains (Secure Debug)
-
-| Debug & Profile | Authenticated debug (Production device) | Required |
-Authorize debug via TEE RoT | Ensure hardware debug prob (e.g., JTAG SWD) is
-disabled in production | Supervisor Domains (Secure Debug)
-
-| Availability | TVM DoS Protection | Required | VMM retains ability to
-interrupt TVM | Prevent TVM from refusing to exit | Supervisor Domains
+==== CoVE-T019 - Malicious TVM causes denial of service
+TVM must not be able to cause a denial of service on the host platform services
+or on other TVMs or ordinary VM workloads.
 
-| Availability | VMM DoS Protection | Implementation-specific | Not in scope
-for CoVE | Prevent non-TCB hosting components from denying service to a TVM |
-Not in scope
+.CoVE-T019
+[options="header"]
+|===
 
-| Side Channel | Address mapping caches (controlled side channel) | Required
-| Supervisor domain Id, MMU, xPMP, MTT | hardware/software TCB should use
-tagging/ partitioning/ flushing techniques to address those types of side
-channels due to temporal/spatial shared resources | Supervisor Domains,
-Security Model
+| Asset | Threat | Adversary | Requirement | Scope | Result
 
-| Side Channel | Transient-execution attack (TEA) side channels |
-Implementation-specific | * Bounds check bypass TEA and variants - should be
-addressed by TVM software using apropos synchronization. Software TCB should use
-synchronization to isolate TCB code from non-TCB code.
-* Branch target injection TEA and variants - should be addressed by software TCB
-via flushing across privilege boundaries to remove untrusted state injected by
-non-TCB software
-* Speculative store bypass TEA and variants- should be addressed by TCB hardware
-via synchronization/barriers to prevent speculative execution of memory reads
-which may allow unauthorized disclosure of information.
-| Implementations should mitigate attacks such as these spectre variants (In
-practice, it is difficult to defend against such attacks in advance) |
-Supervisor Domain Id, additional recommendations in Security Model
+| TVM
+| Disclosure
+| Privileged host software
+| Execution State confidentiality
+| In scope
+| Host-controlled (logical or physical) component reads TVM
+  confidential execution state
```

Comment:
I believe this does not apply to this specific threat. For example, the threatened assets are the host platform services and the threat is the TVM.